lntest/harness.go

```diff
@@ -403,7 +403,9 @@ func (n *NetworkHarness) connect(ctx context.Context,
 tryconnect:
 	if _, err := a.ConnectPeer(ctx, req); err != nil {
 		// If the chain backend is still syncing, retry.
-		if err == lnd.ErrServerNotActive {
+		if strings.Contains(err.Error(), lnd.ErrServerNotActive.Error()) ||
```

Comment:
would `err == lnd.ErrServerNotActive` never be true?

Reply:
It seems so, yes. The error was not in the itest error whitelist so I assume this never actually happened and therefore this was never correct.